common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java

```diff
@@ -262,6 +263,28 @@ public void loadAvatar(@LuaNotNil String playerUUID) {
         NetworkStuff.getUser(new UserData(uuid));
     }
 
+    @LuaWhitelist
+    @LuaMethodDoc(
+        overloads = {
+            @LuaMethodOverload(
+                argumentTypes = {String.class},
+                argumentNames = {"playerUUID"}
+            )
+        },
+        value = "goofy.get_player_color"
+    )
+    public String getPlayerColor(String playerUUID) {
```

Comment:
i feel this should be more
```suggestion
    public String getAvatarColor(String playerUUID) {
```
(acceptance is optional)